examples/shouldi/shouldi/project/cli.py

```diff
@@ -1,12 +1,34 @@
 import pathlib
+from typing import List
 
-from dffml import CMD, Arg
+from dffml import CMD, config, field
 
 from .bom.make import mkbom
 from .bom.db.base import DependencyDB
 from .bom.db.yaml import YAMLDB
 
 
+@config
+class ProjectCreateCMDConfig:
+    source: pathlib.Path = field(
+        "Path to directory containing source code of project",
+    )
+    _dbs: List[DependencyDB.load] = field(
+        "Databases to search for info on dependencies",
+        default_factory=lambda: [],
+    )
+    _add: List[DependencyDB.load] = field(
+        "YAML files containing info to supplement or override auto-detected info",
+        default_factory=lambda: [],
+    )
+    _authoritative: List[pathlib.Path] = field(
```

Comment:
Why are these prefixed with underscores?

Reply:
I prefixed them with underscores to so that they get converted to double dash arguments in cmd.py
I don't know if it's the best solution though. Or we can make them single dash arguments?

Reply:
Let's just make them single dash